lib/documents.js

```diff
@@ -2248,12 +2248,17 @@ Documents.prototype.writeAll = function writeAllDocuments(options) {
       this, options
   );
 };
-function writeAllDocumentsImpl(options) {
+function writeAllDocumentsImpl(jobOptions) {
+
+  if(!jobOptions) {
+    throw new Error('jobOptions cannot be null or undefined.');
+  }
```

Comment:
The story claims that jobOptions can be optional.  Line 2278 tolerates job options that are optional.  

Is the check on line 2253 - 2255 appropriate?